components/dashboard/src/dedicated-setup/DedicatedSetup.tsx

```diff
@@ -78,8 +80,9 @@ const DedicatedSetupSteps: FC<DedicatedSetupStepsProps> = ({ org, config }) => {
     }, [dropConfetti]);
 
     const handleEndSetup = useCallback(() => {
+        markCompleted();
```

Comment:
Q: I recall the question about the session which is replaced at the end of a completed OIDC flow. Do we still need to reset some caches as the ID of the user is about to change on completion of the onboarding?

Reply:
Yah, I've added that [here](https://github.com/gitpod-io/gitpod/pull/17488/files#diff-658b622be42a8e93d942c1349dfc9ff9a996dff6f72a940ac81a126afd8e0b21R84).